src/Console/ConsoleIo.php

```diff
@@ -297,11 +297,10 @@ public function success(array|string $message, int $newlines = 1, int $level = s
      *
      * @param string $message Error message.
      * @param int $code Error code.
-     * @return void
-     * @psalm-return never-return
+     * @return never
```

Comment:
Will this part break php-storm or have they caught up?

Reply:
If PhpStrom doesn't support valid PHP 8.1 return types then it's really not our problem :)

Reply:
https://php.watch/versions/8.1/never-return-type